components/aggregation/Debug.js

```diff
@@ -17,9 +17,23 @@ Bold.propTypes = {
   children: PropTypes.any,
 }
 
-export const Debug = ({ query, children, ...rest }) => {
-  const { query: queryCtx, apiResponse, others, preReshapeTimeRef, reshapeTimeRef, renderTimeRef} = useDebugContext()
+const getCircularReplacer = () => {
+  const seen = new WeakSet()
```

Comment:
Should we use a polyfill for this? According to caniuse this feature is missing in ~6% of web users browsers: https://caniuse.com/?search=Weakset.

Reply:
Also this function seems to be duplicated over here: https://github.com/ooni/explorer/blob/ux/mat-table-view/pages/search.js#L65

Reply:
I believe `next.js` includes polyfill for old browsers. This is in a less frequented code path. But I will double check.

Reply:
1. It looks like WeakSet is polyfilled by `next.js`. I found it in the file `.next/static/chunks/polyfills.js`.
2. This code is in a dev only part (Debug.js) which will not eventually go into production. The other instance in `pages/search.js` is dead code anyway.